tofhir-server/src/main/scala/io/tofhir/server/util/CsvUtil.scala

```diff
@@ -79,34 +80,43 @@ object CsvUtil {
         }
       }
 
+
     // here an example existingContent looks like:
     // [
-    //   [ "header1" -> "value1", "header2" -> "value2" ],
-    //   [ "header1" -> "value3", "header2" -> "value4" ]
+    //   [ "header1" -> "value1", "header2" -> "value2", "header3" -> "value3" ],
+    //   [ "header1" -> "value4", "header2" -> "value5", "header3" -> "value6" ]
     // ]
     //
     // an example newHeaders looks like:
-    // [ "header2", "header3" ]
+    // [ "header2", "headerChanged", "header4" ]
```

Comment:
newHeaders is not a list of string anymore. Please update the example accordingly to prevent any confusion.